bokehjs/src/coffee/common/embed.coffee

```diff
@@ -8,6 +8,18 @@ HasProperties = require "./has_properties"
 {pull_session} = require "./client"
 {Promise} = require "es6-promise"
 
+_handle_notebook_comms = (msg) ->
+  logger.debug("handling notebook comms")
+  # @ is bound to the doc
+  @apply_json_patch_string(msg.content.data)
+
+_init_comms = (target, doc) ->
+  logger.info("Initializing Jupyter comms for target #{target} and doc #{doc}")
+  comm_manager = Jupyter.notebook.kernel.comm_manager
```

Comment:
Just a comment, eventually the `Jupyter` namepace would/could be eliminated (not sure if it will happen or when) and you will need to require Jupyter from namespace...
```
define([
    'base/js/namespace'
], function(Jupyter) {```
```
As I said, the elimination is proposed (in fact you currently can load Jupyter like I show above) but not sure when will happen and if it will happen at the end...